Sources/Plasma/PubUtilLib/plStatusLog/plStatusLog.cpp

```diff
@@ -549,66 +513,50 @@ void    plStatusLog::Bounce( uint32_t flags)
 
 //// IPrintLineToFile ////////////////////////////////////////////////////////
 
-bool plStatusLog::IPrintLineToFile( const char *line, uint32_t count )
+bool plStatusLog::IPrintLineToFile(const ST::string& line)
 {
     if( fFlags & kDontWriteFile )
         return true;
 
-#ifdef PLASMA_EXTERNAL_RELEASE
-    uint8_t hint = 0;
-    if( fFlags & kAppendToLast )
-    {
-        hint = (uint8_t)fSize;
-    }
-#endif
-
     if (!fFileHandle)
         IReOpen();
 
     bool ret = (fFileHandle != nullptr);
 
     if (fFileHandle != nullptr)
     {
-        char work[256];
         ST::string_stream buf;
 
-        //build line to encrypt
+        //build line to write to log file
 
-        if( count != 0 )
+        if (!line.empty())
         {
             if ( fFlags & kTimestamp )
             {
-                snprintf(work, std::size(work), "(%s) ", plUnifiedTime(kNow).Format("%m/%d %H:%M:%S").c_str());
-                buf.append(work);
+                buf << "(" << plUnifiedTime(kNow).Format("%m/%d %H:%M:%S") << ") ";
```

Comment:
```suggestion
                buf << '(' << plUnifiedTime(kNow).Format("%m/%d %H:%M:%S") << ") ";
```

Very minor optimization - using a single character instead of a string so that `ST` doesn't have to calculate the length of the string literal.